### DISCUSSION TITLE
BramMesh, is there a way to combine several surfaces?

### DISCUSSION THREAD

**Discussion by SergioP1975** [nextfoam/baram#137] [Q&A]
I´m meshing a compressor valve (is a cuarter of the model, I will use symmetry) with more than 100 surfaces, mostly of them are "wall", is there a way to combine them? If I play with the Split Feature angle at the importation, some surfaces that I don´t want to combine (symmetry) gets combined... so is there a another way to combine it? The same for outlet/inlet, I have several that would be great if we could combine in only one.

Best regards, and thanks in advance.

**Comment by kdg5424** [comment]
Hello @SergioP1975. This is Donggyu from NEXTFOAM.
For controlling, managing boundary in current status, you can do in Split Surface in 1st step of baramMesh only.
I'll discuss with developers and consider to implement it. (I mean combine or merge boundary)

Best regards

Donggyu Kim

**Reply by kdg5424** [reply]
Hi @SergioP1975. I misunderstood that there is a function which can merge or combine boundary in ANSYS Fluent.
First of all, I'll check it whether it exsits and discuss it with our developers.

**Reply by SergioP1975** [reply]
Thanks for taking in count this feature that would help a lot for complex parts.

**Comment by MS-0123** [comment]
Hello,
there are any news regarding this topic?
Also I'm agree it's very difficult or rather impossible to mesh complex geometry without the possibility to combine several surfeces in main groups.
thanks a lot

**Comment by arniendf** [comment]
For me, this is also currently the biggest challenge ! 

Trying to import CAD models (even rather simple ones) with multiple surfaces & attempting to combine them into a single entity is impossible with some geometry. If this is not possible it becomes tedious to match interfaces in CHT analysis. 

I look forward to the day when a feature is included to make this easier.

**Comment by kdg5424** [comment]
Hello guys. Sorry for the late reply.
We're currently developing someting new with geometry step!!
We'll upload some video on our youtube online channel in this month.
After uploading, I'll let you know.

Best Regards

Donggyu Kim
